### ISSUE TITLE
Adding Linux support

### ISSUE THREAD

**Issue by ashcrow** [botherder/go-autoruns#3]
Adding Linux support will be somewhat interesting. There are two (and a half) popular init systems which would match autorun:

- systemd: Seemingly the most used at this time
- sysvinit: The previous most commonly used and still used often today
- bsd init: The "half". As the name implies it's used in BSD OS's but does show up in some Linux distributions

Of the three I **think** systemd would be the easiest to support. It uses a configuration file format which resembles ``ini`` which is pretty easily parsed.

One way to go about this would be to implement one (IMHO systemd) and add others as time goes on. Once a second init system is supported by ``go-autoruns`` the Linux library will do a check to for each support init system and, if it's in use, parse and add it to the list to return. 

WDYT?

**Comment by botherder** [comment]
I agree, starting point is better than nothing. However what worried me all along implementing Linux support is that there are so many ways that automatically launch things that it would be hard to have comprehensive coverage. Think of rc.local, .xinitrc, .profile, etc.

Ideally it would be good to have a decent overview of what is most commonly used by *nix malware. I guess thinking mostly of all the Mirai and similar.

**Comment by botherder** [comment]
Added support for systemd from @ashcrow. What else should we add?
